guides/common/modules/ref_log-file-directories.adoc

```diff
@@ -1,24 +1,21 @@
-[id='log-file-directories-provided_{context}']
-[id='log-file-directories-provided-by-satellite_{context}']
+[id="Log_File_Directories_{context}"]
 = Log File Directories Provided by {Project}
 
 {ProjectName} provides system information in the form of notifications and log files.
 
-[[tabl-Administering-Logging_and_Reporting-Log_Files_for_Reporting_and_Troubleshooting]]
-
 .Log File Directories for Reporting and Troubleshooting
 [options="header"]
 |===
 | Log File Directories | Description of Log File Content
 | `/var/log/candlepin` | Subscription management
-| `/var/log/foreman` | Foreman
+| `/var/log/foreman-installer` | Installer
 | `/var/log/foreman-proxy` | Foreman proxy
+| `/var/log/foreman` | Foreman
 | `/var/log/httpd` | Apache HTTP server
-| `/var/log/foreman-installer` | Installer
+| `/var/log/messages` | Various other log messages
 | `/var/log/puppet` | Configuration management
```

Comment:
Unrelated, but this hasn't existed since Puppet 4:
```suggestion
| `/var/log/puppetlabs/puppet` | Configuration management
```

Reply:
Applied.